include/deal.II/fe/fe_tools_interpolate.templates.h

```diff
@@ -180,7 +177,7 @@ namespace FETools
               Assert(cell1->at_boundary(face) ||
                        cell1->neighbor(face)->level() == cell1->level(),
                      ExcHangingNodesNotAllowed());
-
+#endif
```

Comment:
I think this got mixed in from another PR.

Reply:
That's from #9619. I'm not sure how this got in here, but if someone merges #9619, I'll rebase this one and these diffs will disappear.